app/controllers/welcome_controller.rb

```diff
@@ -1,8 +1,7 @@
 class WelcomeController < ApplicationController
+
   def show
     locations = Location.all
     @locations = locations.order(:name)
-    #the list should be a filter and the list should be pulled from
-    #a possible google maps api list
   end
```

Comment:
These comments I could actually see keeping. Very informative and I could see myself forgetting to scope these locations on my own.